File: TRBusinessLayer/DataObjects/CarbonFuelTaxWrapper.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using TRBusinessLayer.DataObjects;

namespace TRBusinessLayer.DataObjects
{
    public class CarbonFuelTaxWrapper : AbstractDto
    {

        public List<CarbonFuelTax> AvailFuelTax { get; set; }
        public List<FuelTaxDtl> AvailFuelTaxDtl { get; set; }
    }
}

File: TRClient/src/app/Entities/CarbonFuelTaxWrapper.ts
import {CarbonFuelTax} from './CarbonFuelTax';
import {FuelTaxDtl}    from './FuelTaxDtl';
import {Awrapper}      from './Awrapper';

export class CarbonFuelTaxWrapper  extends Awrapper {
  
    AvailFuelTax: CarbonFuelTax[];
    AvailFuelTaxDtl: FuelTaxDtl[];
    VendorNo: string;
    TaxId: string;
    PaymentMethod: string;
    DueDate: Date;
    TotalLitresToPay: number;
    TotalToPay: number;
   
 }